### PR TITLE
Add travis-ci configuration

### PR DESCRIPTION
**Pull request by coroa**
- Runs tests
- Deploys new tags on master to PyPI

@fabianhofmann:
I activated travis on atlite. So i'd suggest, merging this into master, then rebasing the `v0.0.3` branch ontop of it. Merging the release branch when it is ready to master.

And then going to https://github.com/PyPSA/atlite/releases and "Draft a new release" to generate the new tag and thus push the release to pypi. If that works I'd add the same to vresutils so that all of you should be able to create new releases.

### CODE REVIEW

**Comment by FabianHofmann** [comment]
Sweet!

**Comment by FabianHofmann** [comment]
seems that pytest is missing in the environment

**Comment by FabianHofmann** [comment]
It crosses my mind that one need an cdsapi key for retrieving the era5 data right? This might as well be a problem

**Comment by fneum** [comment]
> It crosses my mind that one need an cdsapi key for retrieving the era5 data right? This might as well be a problem

We can try encrypted variables, but it's differs from the way the CDS API keys are stored in `.cdsapirc`: https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - PYTHON_VERSION="3.7"
+  - PYTHON_VERSION="3.8"
+
+before_install:
+  - wget https://raw.githubusercontent.com/trichter/conda4travis/latest/conda4travis.sh -O conda4travis.sh
+  - source conda4travis.sh
+
+install:
+  - conda config --add pinned_packages python=$PYTHON_VERSION
+  - conda create -n atlite python pip wheel
```
Comment (fneum) [inline .travis.yml:18]:
```suggestion
  - conda create -n atlite python pip wheel pytest
```
We can try this. I think I'd prefer this over an extra `environment.dev.yaml`.

Comment (fneum) [reply .travis.yml:18]: Ah, or just add it to the `environment.yaml`.

**Comment by fneum** [comment]
Here's some details on using `cdsapi` with travis: https://github.com/bluegreen-labs/ecmwfr/issues/3

**Comment by FabianHofmann** [comment]
great thanks!

File: .travis.yml (hunk 1 above)
Comment (coroa) [reply .travis.yml]: > Ah, or just add it to the environment.yaml.

Yes, that was the essence of the third commit.

**Pull request merged**